docker-compose/monitor/Makefile

```diff
@@ -14,8 +14,8 @@ build: clean-jaeger
 # Note: the jaeger "dev" image can be built with "make build".
 .PHONY: dev
 dev: export JAEGER_IMAGE_TAG = dev
-dev:
-	docker compose -f docker-compose.yml up $(DOCKER_COMPOSE_ARGS)
+dev: build-v2
```

Comment:
let's not replace the existing behavior, v2 is not ready for prod yet. You can create `dev-v2` target instead.

Reply:
Understood.